_test/test_sqw/test_sqw_dnd_eval.m

```diff
@@ -202,8 +186,7 @@ function test_sqw_eval_dnd(obj)
             elseif sz(2) ~=1
                 error('FUNC_EVAL:runtime_error','incorrect shape of input arrays');
             end
-            dis = ones(sz);
-            dis(1) = numel(x);
+            dis = 2.*ones(sz);
         end
```

Comment:
such a bed practice for chaning test data. The previous implementation was verifying if it picks up all data, not a partial one. 
if you leave it as it was before, I would see it works as before. Now I do not know without running the code

Reply:
The function relying on the size of the data doesn't work for file-backed vs. memory-backed because the data are not all loaded at once